config-model/src/main/java/com/yahoo/searchdefinition/processing/ImplicitSummaries.java

```diff
@@ -29,17 +29,16 @@ public ImplicitSummaries(Schema schema, DeployLogger deployLogger, RankProfileRe
 
     @Override
     public void process(boolean validate, boolean documentsOnly) {
-        DocumentSummary defaultSummary = schema.getSummary("default");
+        DocumentSummary defaultSummary = schema.getSummariesInThis().get("default");
         if (defaultSummary == null) {
             defaultSummary = new DocumentSummary("default", schema);
-            defaultSummary.setFromDisk(true);
+            defaultSummary.setFromDisk(true); // TODO: Not necessarily
```

Comment:
It is necessary I think as you get the document id with default. So it will access disk.

Reply:
True